posthog/models/event/events_query.py

```diff
@@ -92,18 +92,18 @@ def run_events_query(
     # prevent accidentally future events from being visible by default
     before = query.before or (now() + timedelta(seconds=5)).isoformat()
     try:
-        timestamp = isoparse(before).strftime("%Y-%m-%d %H:%M:%S.%f")
+        parsed_date = isoparse(before)
```

Comment:
nit: couldn't follow why the change from timestamp to parsed date was necessary and am going to assume this was simply a small refactor

Reply:
yep, a small refactor. now constants can also be `datetime`-s, so this leverages that, instead of requiring all "hogql users" to write these long timestamp strings `"%Y-%m-%d %H:%M:%S.%f`